README.md

```diff
@@ -1,27 +1,48 @@
 # What is Che server
-Che Server is a core component of the [Eclipse Che](https://github.com/eclipse/che/). This component is responsible for creation and managing of Che workspaces, but will some day be replaced by the [Dev Workspace Operator](https://github.com/devfile/devworkspace-operator). 
+Che Server provides an Api to retrieve devfile content from the given repositories such as GitHub GitLab Bitbucket and Azure.
+Apart from that Che Server provides an Api to provision Kubernetes namespaces.
```

Comment:
@vinokurig let's add it to the first sentence e.g. `Che Server provides an API for managing Kubernetes namespaces and retrieving  devfile content from repositories hosted on GitHub, GitLab, Bitbucket, and Microsoft Azure Repos`

Reply:
done